index.css

```diff
@@ -29,6 +29,54 @@ h2 {
     font-size: 30px;
 }
 
+/***BUTTON***/
+.btn {
```

Comment:
As far as I could tell, all of the button elements have the btn class. Could the css just be styling the button element instead of a class?